app_pipeline_capabilities.go

```diff
@@ -0,0 +1,96 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"gopkg.in/nullstone-io/go-api-client.v0/response"
+	"gopkg.in/nullstone-io/go-api-client.v0/types"
+	"net/http"
+)
+
+type AppPipelineCapabilities struct {
+	Client *Client
+}
+
+func (e AppPipelineCapabilities) basePath(stackId, appId int64) string {
+	return fmt.Sprintf("orgs/%s/stacks/%d/apps/%d/capabilities", e.Client.Config.OrgName, stackId, appId)
+}
+
+func (e AppPipelineCapabilities) capPath(stackId, appId, capId int64) string {
+	return fmt.Sprintf("orgs/%s/stacks/%d/apps/%d/capabilities/%d", e.Client.Config.OrgName, stackId, appId, capId)
+}
+
+// List - GET /orgs/:orgName/stacks/:stackId/apps/:app_id/capabilities
+func (e AppPipelineCapabilities) List(stackId, appId int64) ([]types.Capability, error) {
+	res, err := e.Client.Do(http.MethodGet, e.basePath(stackId, appId), nil, nil, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	return response.ReadJsonVal[[]types.Capability](res)
+}
+
+// Get - GET /orgs/:orgName/stacks/:stackId/apps/:app_id/capabilities/:id
+func (e AppPipelineCapabilities) Get(stackId, appId, capId int64) (*types.Capability, error) {
+	res, err := e.Client.Do(http.MethodGet, e.capPath(stackId, appId, capId), nil, nil, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	return response.ReadJsonVal[*types.Capability](res)
```

Comment:
It's best to use `ReadJsonPtr[types.Capability]` for struct-types and `ReadJsonVal` for slices,maps.
```suggestion
	return response.ReadJsonPtr[types.Capability](res)
```